### PR TITLE
Fix angular typescript

### PR DESCRIPTION
**Pull request by pcvonz**
Makes sure Typescript files aren't in the directory we publish to NPM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +22,11 @@
     "download:devo": "curl https://otr-backend-service-us-devo.offtherecord.com/v2/api-docs -o api/otr-service.json",
     "clean-source": "find dist/typescript* -type f | grep -v  \"\\.d\\.ts\\|\\.js\" | xargs rm",
     "generate:devo": "npm run generate:devo-angularjs",
-    "generate:devo-angularjs-ci": "npm run download:devo && gulp codegen && java -jar /opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i api/otr-service.json -o dist/typescript -g typescript-angularjs && tsc --project tsconfig.json && npm run clean-source",
+    "generate:devo-angularjs-ci": "npm run download:devo && gulp codegen && java -jar /opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i api/otr-service.json -o myclient/typescript -g typescript-angularjs && tsc --project tsconfig.json --outDir dist/typescript",
```
Comment (Morkos) [inline package.json:25]: 👍

File: tsconfig.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2015",
```
Comment (Morkos) [inline tsconfig.json]: we're in 2024, but any issues with browser compatibles going from es5? I imagine it would be minuscule

**Review by Morkos — APPROVED** [review]
Conditionally approved. See es2015 question below

File: tsconfig.json (hunk 2 above)
Comment (pcvonz) [reply tsconfig.json]: Good question, this change might not be necessary though. I was accidentally generating the `angular` client. Not the `angularjs` client. May as well leave it the same.

Comment (pcvonz) [reply tsconfig.json]: I'll do a little research though.

Comment (pcvonz) [reply tsconfig.json]: Typescript docs say es6 is the way to go!

I also switched from `commonjs` to `es6`, which is recommended by webpack .

**Review by Morkos — APPROVED** [review]

**Pull request merged**